src/main/java/tatracker/logic/commands/session/FilterModuleCommand.java

```diff
@@ -0,0 +1,52 @@
+package tatracker.logic.commands.session;
```

Comment:
Actually, we don't need this command but since you've already made it we'll keep it!